homeassistant/components/assist_satellite/entity.py

```diff
@@ -274,6 +274,10 @@ async def async_internal_start_conversation(
 
         try:
             await self.async_start_conversation(announcement)
+        except Exception:
+            # Clear prompt on error
+            self._extra_system_prompt = None
```

Comment:
Also conversation ID.
```suggestion
            self._conversation_id = None
            self._extra_system_prompt = None
```